css/80_app.css

```diff
@@ -5706,6 +5706,7 @@ li.hide + li.version .badge .tooltip .popover-arrow {
     flex-direction: column;
     background: #fff;
     border-radius: 4px;
+    flex-direction: row;
```

Comment:
This would unconditionally change the menu to be horizontal, whereas https://github.com/openstreetmap/iD/issues/10495#issue-2581547423 suggested only doing so when showing the street-level imagery pane. Most likely, this would clash with the tooltip that appears when hovering over each menu item. I think changing the orientation _could_ be an improvement, but it needs more thought. I don’t know yet whether users would be surprised by a menu that changes from horizontal to vertical depending on what’s around it. This reorientation also doesn’t really solve the problem stated by @danieldegroot2, because you could right-click on a location close to the pane and it would still underlap the pane.

A fix for #10495 should focus on bringing the context menu and its tooltip higher in the z order than the street-level imagery pane.